tools/targets/NCS.py

```diff
@@ -211,5 +238,4 @@ def add_fib_at_start(arginput):
     output_hex_file.merge(input_hex_file, overlap='error')
 
     # Write out file(s)
-    output_hex_file.tofile(file_name_hex, 'hex')
-    output_hex_file.tofile(file_name_bin, 'bin')
+output_hex_file.tofile(file_name_hex, 'hex')
```

Comment:
I think there is a merge error here that is causing everything to fail.  At lease nothing builds until making a change here.  Please add 4 spaces in front of this line to align it with the rest of the function.   
- output_hex_file.tofile(file_name_hex, 'hex')
+    output_hex_file.tofile(file_name_hex, 'hex')